README.md

```diff
@@ -135,6 +135,7 @@ Name | Website | Region
 [Codea IT](/company-profiles/codea-it.md) | https://www.codeait.com | Worldwide
 [CodePen](/company-profiles/codepen.md) | https://codepen.io | Worldwide
 [CodeSandbox](/company-profiles/codesandbox.md) | https://codesandbox.io | Worldwide
+[Coodesh](/company-profiles/coodesh.md) | https://coodesh.com/ | Brazil
```

Comment:
This should be above `[Core-Apps](/company-profiles/core-apps.md) | https://www.core-apps.com/ | USA`